src/test/java/org/openstreetmap/atlas/checks/validation/linear/edges/SingleSegmentMotorwayCheckTest.java

```diff
@@ -24,23 +25,48 @@ public void invalidMotorwaySegmentOneConnectionRoundaboutTest()
     {
         this.verifier.actual(this.setup.invalidMotorwaySegmentOneConnectionRoundaboutAtlas(),
                 new SingleSegmentMotorwayCheck(ConfigurationResolver.emptyConfiguration()));
-        this.verifier.globallyVerify(flags -> Assert.assertEquals(1, flags.size()));
+        this.verifier.globallyVerify(flags ->
+        {
+            Assert.assertEquals(1, flags.size());
+            Assert.assertEquals(1, flags.get(0).getFixSuggestions().size());
+            flags.get(0).getFixSuggestions()
+                    .forEach(s -> Assert.assertTrue(
```

Comment:
Again, please use a more descriptive variable name than a single character.

Reply:
Renamed to fixSuggestion.